tests/dicom/test_dicom_utils.py

```
@@ -0,0 +1,175 @@
+from pathlib import Path
+
+import pytest
+from rich.tree import Tree
```

Ruff

`rich.tree.Tree` imported but unused

Remove unused import: `rich.tree.Tree`

(F401)